### DISCUSSION TITLE
Is there a way to obtain data from `.get_log()`, while at the same time suppressing stdout from `.evolve()`?

### DISCUSSION THREAD

**Discussion by caranha** [esa/pygmo2#164] [Q&A]
Hello! 

I am trying to run some of the algorithms from pygmo a few times, and then produce runtime analysis graphs from those runs. For that, I use the information from `.get_log()`. 

However, I do not need the stdout output from `.evolve()` for all those runs, and I'd like to know if it is possible to suppress that while still getting access to the data from `.get_log()`. If I `set_verbosity()` to zero, I lose both `std_out` and `get_log`. I couldn't find anything that solved my problem in the documentation... 

(I'd prefer not to suppress stdout at the terminal level, because I'm planning to use tqdm to show a progress bar as I go through the algorithm/problem pairs.)

Any ideas?

Thank you!

```
# Setup the run
algo = pg.algorithm(uda)
algo.set_verbosity(10)
pop = pg.population(prob, size = popsize)

pop = algo.evolve(pop)

# Extract the data
log = algo.extract(type(uda)).get_log()
```

**Comment by juan11iguel** [comment]
Would love to find a solution for this as well, currently I am running a lot of evaluations in parallel together with some `tqdm` progress bars and they get hidden from all the evolution logs.
